packages/utils/typescript/lib/generators/index.js

```diff
@@ -1,7 +1,122 @@
 'use strict';
 
-const generateSchemasDefinitions = require('./schemas');
+const path = require('path');
+const chalk = require('chalk');
 
-module.exports = {
-  generateSchemasDefinitions,
+const { TYPES_ROOT_DIR, GENERATED_OUT_DIR } = require('./constants');
+const { saveDefinitionToFileSystem, createLogger, timer } = require('./utils');
+const generateContentTypesDefinitions = require('./content-types');
+const generateComponentsDefinitions = require('./components');
+
+const GENERATORS = {
+  contentTypes: generateContentTypesDefinitions,
+  components: generateComponentsDefinitions,
 };
+
+/**
+ * @typedef GenerateConfig
+ *
+ * @property {object} strapi
+ * @property {boolean} pwd
+ * @property {object} [artifacts]
+ * @property {boolean} [artifacts.contentTypes]
```

Comment:
Wait, are we going with artifacts or artefacts? I don't care either way but let's standardi(s|z)e it :)

Reply:
I swear I saw `artefacts` in the files I just viewed but now I can't see them anymore. Maybe I'm just losing my mind 😆